tests/main/snapd-snap/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: Ensure snapd builds as a snap
+
+# this is what the snapcraft builder on launchpad uses
+systems: [ubuntu-16.04-64]
+
+execute: |
+    echo "Installing snapscraft"
+    apt install -y snapcraft
```

Comment:
Do you need to undo this?

Reply:
same question,  also  do we know if the builder use this or the snapcraft snap?